src/code/views/node-view.coffee

```diff
@@ -165,15 +165,18 @@ module.exports = NodeView = React.createClass
 
   renderSliderView: ->
     enabled = not @props.running or @props.data.canEditValueWhileRunning()
+    showHandle = @props.data.canEditInitialValue()
+    value = @props.data.currentValue or @props.data.initialValue
```

Comment:
The annoyingly-common JS bug: zero will count as false here, so if you have a dependent node and you bring it all the way down to zero, you'll see the fill slider value pop back up to initialValue.

Suggest change to `value = @props.data.currentValue ? @props.data.initialValue`